lib/hanami/validations.rb

```diff
@@ -204,7 +204,7 @@ def messages(type)
       #   class Signup
       #     include Hanami::Validations
       #
-      #     messages :i18n
+      #     messages_path 'config/messages.yml'
```

Comment:
@cllns @artofhuman @solnic It isn't required. `:yaml` is the default (as in `dry-validation`).